game-core/src/main/java/games/strategy/engine/chat/ChatFloodControl.java

```diff
@@ -8,9 +8,10 @@
  * sends more than "N" messages will be filtered until the next time window begins.
  */
 class ChatFloodControl {
-  private static final int ONE_MINUTE = 60 * 1000;
   static final int EVENTS_PER_WINDOW = 20;
+  private static final int ONE_MINUTE = 60 * 1000;
   static final int WINDOW = ONE_MINUTE;
```

Comment:
This is an exception to the decreasing order of accessibility rule.  DeclarationOrder permits a field with higher accessibility to appear after a field with lower accessibility if the initialization of the former depends on the latter, as it does in this case.